### PR TITLE
doc: Created Image Label + Comments Doc

### PR DESCRIPTION
**Pull request by ashokdevtron**
Requested by Badal: https://discord.com/channels/769482988882493450/865151043839197186/1250423878216318976

### CODE REVIEW

File: docs/user-guide/deploying-application/image-labels-and-comments.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Applying Labels and Comments
+
+## Introduction
+
+Typically in a CI pipeline, you build [container images](../../reference/glossary.md#image), and the number of images gradually increases over a period of time.
+
+Assume `azurecr.io/web-server` is your container [image repository](../../reference/glossary.md#repo) and it has multiple images. Now, differentiating one `web-server` image from another or remembering a specific `web-server` image from the repository might become a tedious task.
```
Comment (abhibhaw) [inline docs/user-guide/deploying-application/image-labels-and-comments.md:7]: I can seperate repositories based on the microservices, seperate repo for `web-server` for what usecases I still need this feature?

**Review by abhibhaw — CHANGES_REQUESTED** [review]
- We need to cover some more usecases or keywords when this may be helpful
- We need to add this as linking to other relevant pages. Like maybe building an image, or deploying, communicating to your team if this image is buggy and shouldn't be used for deployment... Or any other precautions required before using this image

File: docs/user-guide/deploying-application/image-labels-and-comments.md (hunk 1 above)
Comment (ashokdevtron) [reply docs/user-guide/deploying-application/image-labels-and-comments.md]: We have now removed the example of repo and mentioned few use cases upfront.

**Review by abhibhaw — DISMISSED** [review]

File: docs/user-guide/deploying-application/image-labels-and-comments.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+You can add labels and comments from the following pages:
+
+* [From Build & Deploy](#from-build--deploy)
+* [From Build History](#from-build-history)
+* [From Deployment History](#from-deployment-history) (only after deployment)
+* [From App Details](#from-app-details) (only after deployment)
+
+{% hint style="warning" %}
+You can add multiple labels to an image. Once added, they cannot be used for another image. <br />
```
Comment (uxarya-d) [inline docs/user-guide/deploying-application/image-labels-and-comments.md:37]: Cannot be used for another image in the application. The label can still be applied to an image in another application.

File: docs/user-guide/deploying-application/image-labels-and-comments.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+2. Empty the content of an existing comment.
+3. Click **Save**.
+
+![Figure 8: Removing a Comment](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/tag-comment/remove-comment.gif)
+
+---
+
+## Extra Use Case
+
+If you use [Application Groups](../application-groups.md) to deploy in bulk, image labels (if added) will be available as filters for you to quickly locate the container image.
```
Comment (uxarya-d) [inline docs/user-guide/deploying-application/image-labels-and-comments.md:110]: Please mention this in Application Group page as well, with a hyperlink to this page.

Comment (uxarya-d) [reply docs/user-guide/deploying-application/image-labels-and-comments.md:110]: Also please make a note of mentioning this in the Release documentation as well

File: docs/user-guide/deploying-application/image-labels-and-comments.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+### From App Details
+
+![Figure 5: Adding Labels and Comments - 'App Details' Page](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/tag-comment/tag-app-details.gif)
+
+---
+
+## Deleting Labels & Comments
+
+### Soft-Delete Labels
```
Comment (uxarya-d) [inline docs/user-guide/deploying-application/image-labels-and-comments.md:61]: We should also mention that soft deleted labels cannot be reused. Only hard deleted labels can be reused. Please verify this.

**Review by uxarya-d — CHANGES_REQUESTED** [review]

File: docs/user-guide/deploying-application/image-labels-and-comments.md (hunk 3 above)
Comment (ashokdevtron) [reply docs/user-guide/deploying-application/image-labels-and-comments.md:110]: Done

File: docs/user-guide/deploying-application/image-labels-and-comments.md (hunk 4 above)
Comment (ashokdevtron) [reply docs/user-guide/deploying-application/image-labels-and-comments.md:61]: Verified. Added as well. Thanks!

File: docs/user-guide/deploying-application/image-labels-and-comments.md (hunk 3 above)
Comment (ashokdevtron) [reply docs/user-guide/deploying-application/image-labels-and-comments.md:110]: Added the second in Masterlist's ToDo sheet

**Review by uxarya-d — APPROVED** [review]

**Pull request merged**